app/services/bookings/gitis/entity.rb

```diff
@@ -15,6 +15,7 @@ module Entity
     include ActiveModel::Model
     include ActiveModel::Attributes
     include ActiveModel::Dirty
+    include ActiveRecord::AttributeMethods::Write
```

Comment:
Looking at the source to this ActiveRecord module, it overwrites a bunch of ActiveModel functionality and I don't think is really intended to be used in this manner with ActiveModel

I'd suggest replacing usage of `write_attribute method_name, value` with 
either `_write_attribute method_name, value`
or `self.attribute= method_name, value`

Either of which don't require the ActiveRecord module?

Reply:
an alternative would be to just replace this line with 

`alias_method :write_attribute, :_write_attribute`

But I recommend either of the former options

Reply:
have implemented `write_attribute` rather than inherit from ActiveRecord, or the other options as the options suggested cause ripple effect errors through out.